pom.xml

```
@@ -3,15 +3,15 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>3.6</version>
+		<version>3.50</version>
 	</parent>
 
 	<artifactId>scriptler</artifactId>
 	<name>Scriptler</name>
 	<version>3.2-SNAPSHOT</version>
 	<packaging>hpi</packaging>
 	<description>Scriptler allows you to store/edit/execute Groovy scripts on any of the slaves/nodes... no need to copy paste Groovy code anymore. Beside administer your scripts, Scriptler also provides a way to share scripts between users via hosted script catalogs on the internet.</description>
-	<url>http://wiki.jenkins-ci.org/display/JENKINS/Scriptler+Plugin</url>
+	<url>https://wiki.jenkins-ci.org/display/JENKINS/Scriptler+Plugin</url>
```

```suggestion
	<url>https://wiki.jenkins.io/display/JENKINS/Scriptler+Plugin</url>
```

Docs should be moved to GitHub tho